docs/en/connector-v2/sink/Redis.md

```diff
@@ -0,0 +1,94 @@
+# Redis
+
+> Redis sink connector
+
+## Description
+
+Used to write data to Redis.
+
+## Key features
+
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [ ] [schema projection](../../concept/connector-v2-features.md)
+
+##  Options
+
+| name      | type   | required | default value |
+|-----------|--------|----------|---------------|
+| host      | string | yes      | -             |
+| port      | int    | yes      | -             |
+| keys      | string | yes      | -             |
```

Comment:
Hi, the naming is a bit odd, keys are usually plural.

Reply:
Emm... Mainly, I want to be consistent with the parameters of the source connector.

Reply:
Update to `key_field`